<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Checklist</class>
 <widget class="QDialog" name="Checklist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>171</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="newButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>New Item</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit Item</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Item</string>
   </property>
  </widget>
  <widget class="QPushButton" name="finishButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Mark Finished</string>
   </property>
  </widget>
  <widget class="QPushButton" name="unfinishButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Mark Unfinished</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>280</y>
     <width>171</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="unfinished">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>0</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Unfinished</string>
   </property>
  </widget>
  <widget class="QLabel" name="finished">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Finished</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lockButton">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>132</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Lock</string>
   </property>
  </widget>
  <widget class="QPushButton" name="unlockButton">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>192</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Unlock</string>
   </property>
  </widget>
  <widget class="QPushButton" name="upButton1">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="downButton1">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Down</string>
   </property>
  </widget>
  <widget class="QPushButton" name="upButton2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="downButton2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Down</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
